### PR TITLE
ztp/entrypoint reshuffle

### PR DESCRIPTION
**Pull request by lack**
- ztp: Add a default 'help' endpoint to the resource-generator container
- ztp: Add a new entrypoint to make extracting directories from the ZTP container easier to do

(Note: Also build on the changes in #925; diffs will be clearer once that merges)

### CODE REVIEW

File: ztp/gitops-subscriptions/argocd/README.md [hunk 1]
```diff
@@ -59,16 +59,29 @@ The following steps prepare the hub cluster for site deployment and initiate ZTP
 - Push your changes to the git repository. The SiteConfig and PolicyGenTemplate CRs may be pushed simultaneously.
 
 ### Monitoring progress
-The ArgoCD pipeline uses the SiteConfig and PolicyGenTemplate CRs in GIT to generate the cluster configuration CRs & ACM policies then sync them to the hub. The progress of this synchronization can be monitored in the ArogCD dashboard.
+The ArgoCD pipeline uses the SiteConfig and PolicyGenTemplate CRs in GIT to generate the cluster configuration CRs & ACM policies then sync them to the hub.
 
-The progress of cluster installation can be monitored from the command line:  
+The progress of this synchronization can be monitored in the ArogCD dashboard.
```
Comment (imiller0) [inline ztp/gitops-subscriptions/argocd/README.md:64]: typo arogCD

File: ztp/gitops-subscriptions/argocd/README.md [hunk 2]
```diff
@@ -59,16 +59,29 @@ The following steps prepare the hub cluster for site deployment and initiate ZTP
...
      $ curl -sk $(oc get agentclusterinstall -n $CLUSTER $CLUSTER -o jsonpath='{.status.debugInfo.eventsURL}')  | jq '.[-2,-1]'
 ```
+
+2. The Topology Aware Lifecycle Operator then applies the configuration policies which are bound to the cluster
+
+Each cluster's policies will be applied in the order defined by the ran.openshift.io/ztp-deploy-wave annotations.
+
 The progress of configuration policy reconciliation can be monitored in the ACM dash board.
 
+The final policy that will become compliant is the one defined in the `du-validator-policy-*` policies. This policy, when compliant on a cluster, corresponds to the ZTP process completing, ensuring that all cluster configuration, operator installation, and operator configuration has completed.
```
Comment (imiller0) [inline ztp/gitops-subscriptions/argocd/README.md:83]: Maybe mention here that TALO will set the ztp-done label on the cluster as well?

Comment (lack) [reply ztp/gitops-subscriptions/argocd/README.md:83]: Does it?  I didn't know if that merged yet...

Comment (imiller0) [reply ztp/gitops-subscriptions/argocd/README.md:83]: It hasn't yet. We should update when it does.

**Review by imiller0 — APPROVED** [review]
/lgtm

**Pull request merged**